packages/ethereum-storage/src/gas-price-definer.ts

```diff
@@ -60,10 +68,13 @@ export class GasPriceDefiner {
       const gasPriceArray = await this.pollProviders(type, network);
       if (gasPriceArray.length > 0) {
         // Get the highest gas price from the providers
-        return gasPriceArray.reduce(
+        const gasPrice = gasPriceArray.reduce(
           (currentMax, gasPrice) => (currentMax.gt(gasPrice) ? currentMax : gasPrice),
           BigNumber.from(0),
         );
+        return this.gasPriceMin && gasPrice.lt(this.gasPriceMin)
+          ? BigNumber.from(this.gasPriceMin)
```

Comment:
It was to clone the object and avoid modifying it in later code. But since BigNumbers is immutable by design, it's useless in that case. I'll remove it.